selfservice/strategy/oidc/strategy_login.go

```diff
@@ -74,6 +74,13 @@ type UpdateLoginFlowWithOidcMethod struct {
 
 	// The identity traits. This is a placeholder for the registration flow.
 	Traits json.RawMessage `json:"traits"`
+
+	// UpstreamParameters are the parameters that are passed to the upstream identity provider.
+	//
+	// These parameters are optional and depend on the upstream identity provider.	// UpstreamParameters are validated against the provider's AllowedUpstreamParameters configuration.
```

Comment:
Maybe replace this text with the actually supported values? I think that helps users a lot :)